addons/binding/org.openhab.binding.omnilink/src/main/java/org/openhab/binding/omnilink/handler/ZoneHandler.java

```diff
@@ -23,6 +23,7 @@
 
 public class ZoneHandler extends AbstractOmnilinkHandler {
     private Logger logger = LoggerFactory.getLogger(ZoneHandler.class);
+    private ZoneStatus zoneStatus;
```

Comment:
It has been a while since I have used it too. :)

We have multiple threads accessing the zoneStatus variable, a memory barrier is required.

With the upgrade of volatile in Java 1.5+, I think this is sufficient.  But to be honest, I am coming back to Java after a 2 year break.  

I will double check tomorrow if this will gaurentee against out of order writes on that field.